content/logs/languages/php.md

```diff
@@ -0,0 +1,562 @@
+            # logs alls events (debug trought fatal)
+            level: debug
+```
+
+#### Laravel
+
+```php
+//file: bootstrap/app.php
+$app->configureMonologUsing(function($monolog) {
+    $monolog->pushHandler(...);
+    
+  // configure your logger as below 
+});
+
+return $app;
+```
+
+#### Silex
+
+```php
+// file: bootstrap
+$app->extend('monolog', function($monolog, $app) {
+    $monolog->pushHandler(...);
+    // configure your logger as below 
+    return $monolog;
+});
+```
+
+#### Lumen
+
+```php
+//file: bootstrap/app.php
+$app->configureMonologUsing(function($monolog) {
+    $monolog->pushHandler(...);
+    // configure your logger as below 
+});
+
+return $app;
+```
+
+#### CakePHP
+
+First, add this dependency to the `composer.json` file and
+run `composer update`.
+
+```json
+{
+    "require": {
+        "cakephp/monolog": "*"
+    }
+}
+```
+
+Then, start by creating a logging configuration file (i.e. `app/Config/log.php`) that you include your `app/Config/bootstrap.php`:
+
+```php
+include 'log.php';
+```
+
+A basic configuration, to replicate what Cake does but using Monolog would look something like this:
+
+```
+CakePlugin::load('Monolog');
+```
+
+Finally log into a file:
+
+```
+CakeLog::config('debug', array(
+  'engine' => 'Monolog.Monolog',
+  'channel' => 'app',
+  'handlers' => array(
+    'Stream' => array(
+      LOGS . 'application-json.log',
+      'formatters' => array(
+        'Json' => array("")
+      )
+    )
+  )
+));
+```
+
+## PHP Symfony
+
+This section is about:
+
+* How to send your logs to Datadog from a Symfony 2 environment
+* How to configure Monolog to send logs in JSON
+* How to enrich your Log events with session contextual data
+
+### Setup - Log into files with Monolog
+
+1. Declare a Monolog JSON formatter as a service:
+
+    ```yaml
+    services:
+        monolog.json_formatter:
+            class: Monolog\Formatter\JsonFormatter
+    ```
+
+2. Wire the formatter in your Monolog configuration: declare the formatter field as follows:
+
+    ```yaml
+     monolog:
+        handlers:
+            main:
+                type:   stream
+                path:   "%kernel.logs_dir%/%kernel.environment%.log"
+                level:  debug
+                formatter: monolog.json_formatter
+    ```
+
+### Configure your Datadog agent
+
+Create a `php.d/conf.yaml` file in your `conf.d/` folder with the following content:
+
+```yaml
+init_config:
+
+instances:
+    
+##Log section
+logs:
+
+    ## - type : file (mandatory) type of log input source (tcp / udp / file)
+    ##   port / path : (mandatory) Set port if type is tcp or udp. Set path if type is file
+    ##   service : (mandatory) name of the service owning the log
+    ##   source : (mandatory) attribute that defines which integration is sending the logs
+    ##   sourcecategory : (optional) Multiple value attribute. Can be used to refine the source attribtue
+    ##   tags: (optional) add tags to each logs collected
```

Comment:
[Same comment](https://github.com/DataDog/documentation/pull/2127/files#r175451646) here about the colon spacing.